tests/lib/tools/suite/to-one-line/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: integration tests for str-tool
+execute: |
+    # Check help
```

Comment:
Could you please add shellcheck and check the variant that invokes the program without any arguments?

Reply:
sure

Reply:
This is not done yet.